Makefile.oss.prow

```diff
@@ -13,28 +13,46 @@
 # limitations under the License.
 
 # special prow related things
+# Images are tagged with an arbitrary semver and git SHA. The git SHA ensures
+# unique tags and makes the images traceable back to a commit. The semver should
+# be revved upon any major changes to the image.
 
-# This is the image that our prow nomos-presubmit-e2e job runs so that it can
-# use KIND instead of setting up a GKE cluster per test run.
+# This is the image that our prow kind/e2e presubmit job runs so that it can
+# use KIND instead of setting up a GKE cluster per test run. Based on kubekins-e2e
+# for docker-in-docker support to build docker images and run kind.
 
 # Note: nothing builds this, if you update the version, you will need to rebuild manually.
-# For the version tag, concatenate the kubekins version and KIND version
-E2E_TEST_IMAGE_OSS_PROW_KUBEKINS_REGISTRY := k8s-staging-test-infra
-E2E_TEST_IMAGE_OSS_PROW_KUBEKINS := v20230309-9a6b1b3121-1.23
-E2E_TEST_IMAGE_OSS_PROW_KIND := v0.14.0
-E2E_TEST_IMAGE_OSS_PROW_TAG := kubekins-e2e-$(E2E_TEST_IMAGE_OSS_PROW_KUBEKINS)-kind-$(E2E_TEST_IMAGE_OSS_PROW_KIND)
-E2E_TEST_IMAGE_OSS_PROW := gcr.io/oss-prow-build-kpt-config-sync/e2e-prow:$(E2E_TEST_IMAGE_OSS_PROW_TAG)
-image-oss-prow: build/prow/kubekins-kind/Dockerfile
-	@echo "+++ Building the image for the oss-prow presubmit run"
+# TODO: setup postsubmit job to build this image
+KUBEKINS_E2E_TAG := v1.0.0-$(shell git rev-parse --short HEAD)
```

Comment:
wont this mean you have to build it for every commit? otherwise the tag wont exist, even if nothing changed

Reply:
I'm mildly concerned people will just re-build this without incrementing the semver. But i guess it's not a big deal

Reply:
That's fair. I'm not as opinionated about the semver, but I'd like to have the git SHA in the tag so that images can be traced back to the commit which built them